run.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
-python scripts/check_requirements.py requirements.txt
-if [ $? -eq 1 ]
-then
+if [ ! -d "./env_check.lock" ];then
```

Comment:
Change the check for the lock file from a directory (-d) to a regular file (-f).
```suggestion
if [ ! -f "./env_check.lock" ];then
```